include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -1759,31 +1761,33 @@ namespace internal
         const std::vector<std::pair<unsigned int, unsigned int>> &cells,
         const std::vector<FaceToCellTopology<VectorizedArrayType::size()>>
           &                                            faces,
+        const std::vector<unsigned int> &              active_fe_index,
         const dealii::hp::MappingCollection<dim> &     mapping_in,
         MappingInfo<dim, Number, VectorizedArrayType> &mapping_info,
         std::pair<
           std::vector<
             MappingInfoStorage<dim - 1, dim, Number, VectorizedArrayType>>,
           CompressedFaceData<dim, Number, VectorizedArrayType>> &data)
       {
-        AssertDimension(mapping_in.size(), 1);
-        const auto &mapping = mapping_in[0];
-
         FE_Nothing<dim> dummy_fe;
 
+        const unsigned int n_active_fe_indices =
+          active_fe_index.size() > 0 ?
+            (*std::max_element(active_fe_index.begin(), active_fe_index.end()) +
+             1) :
+            1;
```

Comment:
Could you find a more descriptive variable name? This is more like `max_active_fe_index_plus_one` but there may be a better one :)

Reply:
Maybe move the `+1` part to the resize operations later, and turn this variable into `max_active_fe_index`.

Reply:
It's more `n_(potentially_locally_relevant_)active_fe_indices` but I don't think we need to be so verbose here.

Reply:
I am with @marcfehling here because this really seems like `max_active_fe_index`. By `n_active_fe_indices` I would think of filtering out lower numbers which does not happen (at least not here). Not that our collections waste many numbers, but I like the name.